docs/dasharo-tools-suite/documentation.md

```diff
@@ -98,7 +54,14 @@ To access Dasharo Tools Suite:
 * attach a wired network cable to the device's Ethernet port,
 * power on the device, holding down the Boot Menu entry key,
 * in the Boot Menu, select the `iPXE Network Boot` option,
-* in the Network Boot menu, select the `Dasharo Tools Suite` option,
+* in the Network Boot menu, select the `Dasharo Tools Suite` option, or enter
+  iPXE shell and type by hand:
+
+    ```bash
+    dhcp net0
+    chain https://boot.3mdeb.com/dts.ipxe
```

Comment:
```suggestion
    chain http://boot.dasharo.com/dts/dts.ipxe
```

Reply:
@macpijan why not https://boot.dasharo.com/dts/dts.ipxe? BTW above one seem to also work fine :thinking:

Reply:
Ok, I see that old iPXE e.g. on v4.19.0.1 does not have https support

Reply:
maybe that should be known limitation

Reply:
yes here we should still recommend https, if on some specific platform https does not work, then user must fall back

Reply:
@macpijan please take a look at this: dacb0133f11d

Reply:
looks fine